### PR TITLE
🚧Add caching to the Font.Measure process

### PR DESCRIPTION
**Pull request by CalvinWilkinson**
<h1 style="font-weight:bold" align="center">Preview Feature Pull Request</h1>
<h2 style="font-weight:bold" align="center">📋Review Checklist📋</h2>

- [ ] The **_head(source)_** branch for this pull request is a **_preview feature_** branch, with the correct naming syntax, in the following example:
  - A **_head(source)_** branch is the branch being merged into a target branch.
    <details closed><summary>Example</summary>

      ``` xml
      Syntax: preview/feature/<issue-num>-<description>
      Example: preview/feature/123-my-preview-feature
      ```
    </details>

- [ ] The **_head(source)_** branch for this pull request is created from a **_preview release_** branch, with the correct naming syntax, in the following example:
  <details closed><summary>Example</summary>

    ``` xml
    Syntax: preview/v<major>.<minor>.<patch>-preview.<prev-num>
    Example: preview/v1.2.3-preview.4
    ```
  </details>

- [ ] The **_base(target)_** branch for this pull request is a **_preview release_** branch, with the correct naming syntax, in the following example:
  - A **_base(target)_** branch is the branch that the **_head(source)_** branch is merging into.

    <details closed><summary>Example</summary>

      ``` xml
      Syntax: preview/v<major>.<minor>.<patch>-preview.<prev-num>
      Example: preview/v1.2.3-preview.4
      ```
    </details>

💡For more information on branching, refer to the project's [branching documentation](https://github.com/KinsonDigital/.github/blob/master/docs/BranchingFullGitFlow.md).

- [ ] Associated issue exists and is linked to this pull request.
  - One issue per pull request.

- [ ] Pull request title matches the title of the associated issue.

- [ ] A `preview` label has been added to the pull request.

- [ ] The labels attached to this PR match the labels attached to the associated issue.

- [ ] Project assigned matches the project of the associated issue.

- [ ] Milestone assigned matches the milestone of the associated issue.

- [ ] My code follows the coding style of this project.
  - The style is enforced by the **_.editorconfig_** files in the project and displayed as warnings.  If there is an edge case, reach out, and let's discuss it.

- [ ] I have manually tested my code changes to the best of my ability.

- [ ] All tests passed locally.
  - This is required because unit tests might pass locally but not in the CI environment during the status check process or vice-versa.
  - Tests might pass on the developer's machine but not necessarily on the code reviewer's machine.
  - Status checks run for this pull request when merging into the **_preview/v\*.\*.\*-preview.\*_** branch.  These status checks run every time a change is pushed to the pull request.  These checks validate version syntax, tagging, builds, unit tests, and more.

### CODE REVIEW

**Comment by CalvinWilkinson** [comment]
> **Note** Added a https://github.com/KinsonDigital/Velaptor/labels/%E2%9C%A8new%20feature
> This feature is a caching feature that can be enabled or disabled for the Font class.

File: Velaptor/Content/Fonts/Font.cs [hunk 1]
```diff
@@ -426,4 +397,64 @@ private void RebuildAtlasTexture()
 
         this.metrics = glyphMetrics;
     }
+
+    /// <summary>
+    /// Returns the size of the given single line of <paramref name="line"/>.
```
Comment (KinsonDigitalAdmin) [inline Velaptor/Content/Fonts/Font.cs:402]: Refactor code docs too

```suggestion
    /// Returns the size of the given single <paramref name="line"/> text.
```

File: Velaptor/Content/Fonts/Font.cs [hunk 2]
```diff
@@ -426,4 +397,64 @@ private void RebuildAtlasTexture()
...
+    /// </summary>
+    /// <param name="line">The line of text to measure.</param>
+    /// <returns>The size of the line of text.</returns>
```
Comment (KinsonDigitalAdmin) [inline Velaptor/Content/Fonts/Font.cs:405]: Refactor code docs

```suggestion
    /// <returns>The size of the <paramref name="line"/>.</returns>
```

File: Velaptor/Content/Fonts/IFont.cs [hunk 3]
```diff
@@ -65,6 +65,16 @@ public interface IFont : IContent
     /// </summary>
     float LineSpacing { get; }
 
+    /// <summary>
+    /// Gets or sets a value indicating whether or not to cache the measurements of the text.
+    /// </summary>
+    bool CacheEnabled { get; set; }
+
+    /// <summary>
+    /// Gets the maximum number of measurements to cache.
```
Comment (KinsonDigitalAdmin) [inline Velaptor/Content/Fonts/IFont.cs:74]: Refactor code docs

```suggestion
    /// Gets or sets the maximum number of measurements to cache.
```

**Review by KinsonDigitalAdmin — CHANGES_REQUESTED** [review]
